<?xml version="1.0"?>
<launch>
    <!--change to your own parameters !!!!!!!!!!!-->
    <param name="intrinsic" value="$(find lidar_camera_fusion)/config/intrinsic_monash.txt" />
    <param name="extrinsic" value="$(find lidar_camera_fusion)/config/extrinsic_monash.txt" />
    <param name="min_distance" value="0.3" />
    <param name="max_distance" value="2.5" />

    <!--change to your own image and lidar path !!!!!!!!!!!-->
    <param name="image_path" value="/home/hwkan/rosbag/lidar_rgb_fusion/test_dataset/image" />
    <param name="cloud_path" value="/home/hwkan/rosbag/lidar_rgb_fusion/test_dataset/cloud" />
    <!--change to your own save path !!!!!!!!!!!-->
    <param name="save_path" value="/home/hwkan/rosbag/lidar_rgb_fusion/test_dataset/fused_cloud" />

    <!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find lidar_camera_fusion)/rviz_cfg/sensor_fusion.rviz" /> -->

</launch>